### ISSUE TITLE
Chat Module Not Working when reCAPTCHA is enabled

### ISSUE THREAD

**Issue by mahavirn** [freescout-helpdesk/freescout#4050]
Chat Module Not Working when reCAPTCHA is enabled "(reCAPTCHA) End-User Portal "Submit a Ticket" Form".

It gives below error:

```
Call to a member function has() on null (View: /freescout/Modules/ExtraSecurity/Resources/views/recaptcha_eup_submit.blade.php) (View: /freescout/Modules/ExtraSecurity/Resources/views/recaptcha_eup_submit.blade.php) (View: /freescout/Modules/ExtraSecurity/Resources/views/recaptcha_eup_submit.blade.php) {"exception":"[object] (ErrorException(code: 0): Call to a member function has() on null (
```

The chat module doesnot load and gives error when this "(reCAPTCHA) End-User Portal "Submit a Ticket" Form" is enabled.

FreeScout Version: `1.8.137`
Extra Security Version: `1.0.9`
Live Chat Version: `1.0.39`

**Comment by freescout-helpdesk** [comment]
We can't reproduce the issue. Do you have modules updated to their latest versions? Are you using any non-official modules or cusmotizations?

Can you send the screenshot of the "(reCAPTCHA) End-User Portal "Submit a Ticket" Form" settings form.

**Comment by mahavirn** [comment]
Yes, all of my modules are updated to latest version, below is the screenshot of the form, removed the values that i have there.

<img width="752" alt="Screenshot 2024-06-01 at 2 30 51 PM" src="https://github.com/freescout-helpdesk/freescout/assets/118246547/798ddadd-fcea-44ca-900f-5e46daedd51a">

**Comment by freescout-helpdesk** [comment]
In `/freescout/Modules/ExtraSecurity/Resources/views/recaptcha_eup_submit.blade.php` try to replace line 50 with:
```
@if ($errors && $errors->has('extrasecurity.recaptha'))
```

**Comment by mahavirn** [comment]
Yes @freescout-helpdesk 
Already did that, Thanks!

Posted here so it should be fixed in next release.

Thanks!

**Comment by freescout-helpdesk** [comment]
Fixed in Extra Security Module v1.0.10
